src/components/Search/SearchResultsListItem.tsx

```diff
@@ -12,22 +12,16 @@ import { truncateHash, getLatestCommitMessage } from '../../utils/helpers';
 function SearchResultsListItem(props: SearchResultsListItemProps) {
   const { index, item } = props;
   const isChecked: boolean = useAppSelector((state) =>
-    state.checkedRevisions.revisions.includes(index),
+    state.checkedRevisions.revisions.includes(item),
   );
   const { handleToggle } = useCheckRevision();
 
-  const indexString = index.toString();
```

Comment:
I think this was added for typechecking lint errors on the string interpolation, but it doesn't appear to be needed anymore.